EduNLP/SIF/sif.py

```diff
@@ -10,39 +10,43 @@
 __all__ = ["is_sif", "to_sif", "sif4sci"]
 
 
-def is_sif(item):
+def is_sif(item, check_formula=True):
```

Comment:
Is sif should only return True or False. If it is necessary to modify the item for reducing duplicate opertaion in the following procedure, an extra argument is expected.